### DISCUSSION TITLE
Affranto!

### DISCUSSION THREAD

**Discussion by Miksca** [indomus/forum#3270] [HUB: Home Assistant]
Ho seguito tutta la procedura per installare [Home Assistant OS] e sono arrivato fino in fondo... quando mi sono connesso a http://homeassistant.local:8123/ mi ha chiesto user e psw che ho messo.
Il sistema sembrava imbambolato e alla fine mi sono ritrovato con una richiesta di login sul browser e un "waiting for supervisor to startup..." sul mini pc HA. se faccio login mi dice che qualcosa è sbagliato... ho rifatto tutta la procedura ma arrivo sempre allo stesso finale..

**Comment by KC-inDomus** [comment]
mi par strano. descrivi meglio ciò che fai

**Comment by Miksca** [comment]
Ciao ho fatto controllare a uno smanettone il mini pc di HA... in verità sul HW non c'è finito su nessun utente è per quello che non riesco a resettare la psw!! il fatto è che via web invece chiede user e psw che in realtà non sono finiti sul HW e a questo punto non ho la minima idea di come fare a resettare.
Sapete come posso fare a resettare http://homeassistant.local:8123/?
Grazie

**Reply by NdR91** [reply]
A parte che se fai mettere mano "all'amico smanettone" continuando a non descriverci esattamente cos'hai fatto, risulta difficile aiutarti.

Poi: 

> Sull'HW non c'è finito nessun utente

Ma cosa significa? Che hardware? 
> Ho seguito tutta la procedura per installare [Home Assistant OS]

Che procedura? Proprio OS o Supervised?
> Se faccio login mi dice che qualcosa è sbagliato

Cosa esattamente?

Hai guardato i log?

**Comment by Miksca** [comment]
Ciao provo a descrivere meglio il problema:
ho seguito la procedura Home Assistant OS
https://indomus.it/guide/installare-e-configurare-home-assistant-os-su-un-mini-pc-intel-nuc-dedicato/
per installare il tutto su [questo mini pc](https://www.amazon.it/gp/product/B0CLCZBVTQ?th=1&linkCode=ll1&tag=indomus-21&linkId=30a3a2bc990365b7533137443dfb0b4d&language=it_IT&ref_=as_li_ss_tl) sono arrivato in fondo e dopo essermi connesso con http://homeassistant.local:8123/
al mini pc è partita correttamente la procedura d'installazione e dalla schermata di benvenuto ho impostato le prime informazioni compreso username e password.
Qui nasce il problema: tutto il sistema si è inchiodato (lato computer d'appoggio) e ho dovuto riavviare.
Riconnettendomi a http://homeassistant.local:8123/ mi chiedeva user e password ma non gli andavano bene.
Ho quindi seguito la procedura per resettare la password dal prompt dei comandi di HA sul mini pc ma non riuscivo in alcun modo.
Qui s'inserisce l'amico "smanettone" che mi controllando mi ha detto che sul mini pc non c'è nessun utente e che con un comando che non vi so riportare c'è solo il "ROOT" e quindi, a suo parere, è impossibile resettare alcunchè non essendoci nessun supervasior registrato.
Adesso sono fermo perchè da a http://homeassistant.local:8123/ mi chiede sempre la user e psw e da prompt dei comandi del mini pc sembra che non ci sia nessun utente.
Grazie della pazienza

**Comment by Miksca** [comment]
se digito auth list mi esce user : [ ]

**Comment by Miksca** [comment]
Penso di aver capito cosa è successo ma non so cosa fare.
Dopo il primo problema con l'account dove non riuscivo a resettare la password ho re-installato tutto rifacendo la procedura Home assistant OS ripartendo dalla chiavetta e ho sicuramente cancellato l'account sul mini pc.
Pensavo che collegandomi di nuovo con  http://homeassistant.local:8123/ riconoscesse che non c'era nessun utente e mi facesse rifare la procedura.
Sembra invece che sia rimasto "appeso" l'account sull'interfaccia web... cosa faccio?? Grazie

**Comment by Miksca** [comment]
Cosa faccio? formatto il disco e riparto da zero con l'installazione  Home Assistant OS?
Forse senza formattare sono rimasti i memoria dati...
Grazie

**Reply by KC-inDomus** [reply]
chi te lo vieta. danni non non ne fai👍

**Comment by Miksca** [comment]
Ce l'ho fatta! riporto nel caso qualche pellegrino come me finisse in questo casino.
Le cose erano come sopra avevo intuito.
Per sistemare ho:
Ripristinato e usato la chiavetta come da procedura Home Assistant OS
Try Ubuntu
Usato comando Disk
selezionato partizioni del disco che si era allocato Hassos
Formattato le partizioni ripristinandole per windows
riseguito la procedura Home Assistant OS dal prompt dei comandi
Sperem....
